linker/Linker/AssemblyUtilities.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Mono.Cecil;
+
+namespace Mono.Linker {
+
+	public class AssemblyUtilities {
```

Comment:
Doesn't need to be public but should be static

Reply:
Made it static, but need to keep it public so I can use it from ILLink.Tasks.